matisse/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,8 @@
     <string name="error_file_type">Unsupported file type</string>
     <string name="error_type_conflict">Can\'t select images and videos at the same time</string>
     <string name="error_no_video_activity">No App found supporting video preview</string>
+    <string name="error_over_original_size">Can\'t select the images larger than %1$d MB</string>
+    <string name="error_over_original_count">%1$d images over %2$d MB. Selection of those images will be removed</string>
```

Comment:
en ,  you can try this feature by run the sample .In fact , when you checked original checkbox ,it will just uncheck the checkbox after the dialog dimissed .so "removed " may not be a apposite word.

Reply:
What is the purpose of "Original" option in the selection screen?

Reply:
let user decide weather use original photo in next step or not .

Reply:
I could see only the URI and File Path on both Original radio button checked and unchecked cases. What do you mean by next step?

Reply:
It will be great, if you tell me simple use case of the Original option.

Reply:
next step means you press sure button ,to use the selected photo in your own project.

case 1: you check original-checkbox, then you select a photo which oversize the original maxSize limit  ,then the origin-checkbox will unchecked and show a dialog.

case 2:  you selected some photos , if one of those photo's size is oversized ,then if you want to check original-checkbox, it will fail with a dailog

Reply:
By use case, i mean what is the difference during the selection when users checked and unchecked? 

After pressing the sure button, whether the Original is checked or unchecked during the selection, i just get the media files URL and URI that's it. What is the purpose of Original option?

Reply:
Matisse.obtainOriginalState(Intent data)   can get Original option info from user.